src/htmlrenderer.cpp

```diff
@@ -482,6 +490,102 @@ void HtmlRenderer::render(std::istream& input,
 				}
 				break;
 			}
+
+			case HtmlTag::VIDEO: {
+				std::string videourl;
+				try {
+					videourl = xpp.get_attribute_value("src");
+				} catch (const std::invalid_argument&) {
+					videourl = "";
+				}
+				video_count++;
+				if (videourl.length() > 0) {
```

Comment:
I guess this was just copied from `HtmlTag::IMG` handler, but it's more idiomatic to write `if (!videourl.empty())`.